buildspec.yml

```diff
@@ -5,7 +5,7 @@ phases:
   install:
     commands:
       - git clone https://github.com/plus3it/terraform-bootstrap.git && chmod +x terraform-bootstrap/install.sh
-      - terraform-bootstrap/install.sh
+      - terraform-bootstrap/install.sh -v 1.0.11
```

Comment:
@eemperor What do you think about using the tardigrade-ci target here? That project is already version-pinned, and each version installs a specific version of terraform. Basically I think we could replace *both* these lines with:

```
      - make terraform/install
```

Reply:
We would need to add `Makefile` configured with a specific version of `tardigrade-ci` right?

Reply:
I think I see what you're getting at.  The Dockerfile with tardigrade-ci is already present.

Reply:
The Dockerfile and GNUmakefile are already present (make will use Makefile or GNUmakefile by default). So I think it would only require that one line to use the make target to install terraform.

Reply:
Okay.  I made the update to use `tardigrade-ci`.

Weird...  I was seeing two copies of my initial comment so proceeded to delete one but now both are gone.  Something with my "internet".

Reply:
Blergh. Tardigrade-ci installs binaries to `$HOME/bin`. So need to add that to the PATH I think, `export PATH=$HOME/bin:$PATH`